apps/bundle_deploy/demo.cc

```diff
@@ -0,0 +1,65 @@
+#include "tvm/runtime/c_runtime_api.h"
+#include <assert.h>
+#include <dlfcn.h> //dlopen
+#include <dlpack/dlpack.h>
+#include <iostream>
+#include <random>
+#include <vector>
+
+template <typename F> auto getFunc(void *bundle, const char *name) {
+  dlerror();
+  auto *f =
+      reinterpret_cast<typename std::add_pointer<F>::type>(dlsym(bundle, name));
+  assert(!dlerror());
+  return f;
+}
+
+int main(int argc, char **argv) {
+  auto *bundle = dlopen(argv[1], RTLD_LAZY | RTLD_LOCAL);
```

Comment:
suggest to print the usage, or describe it in README.

Reply:
Done, thanks for the suggestion.